keyboards/handwired/twadlee/tp69/rules.mk

```diff
@@ -0,0 +1,26 @@
+# MCU name
+MCU = MKL26Z64
+USE_CHIBIOS_CONTRIB = yes
+
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```

Comment:
```suggestion

```

Reply:
Here I was following `keyboards/handwired/onekey/teensy_lc`: https://github.com/qmk/qmk_firmware/blob/4c4009c0fd3e55c2eacde302ef08e56a3ff2ab60/keyboards/handwired/onekey/teensy_lc/rules.mk#L6

Teensy LC support requires ChibiOS-Contrib.

Reply:
These suggestions just move it to the bottom.

Reply:
Didn't see the second half of the diff.